ant-protocol/src/error.rs

```diff
@@ -18,24 +18,11 @@ pub type Result<T> = std::result::Result<T, Error>;
 #[derive(Error, Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
 #[non_exhaustive]
 pub enum Error {
-    // ---------- Misc errors
-    #[error("Could not obtain user's data directory")]
-    UserDataDirectoryNotObtainable,
```

Comment:
removal of this will cause disorder when try to parse old `Error::ChunkDoesNotExist` (entry index 4) into new (where entry index 4 is `QuoteGenerationFailed` )   ?
which won't fall back to `unkown` ?

Reply:
I'm not sure that is the case, we've been changing this struct for a while now, so I thought it was time to clean that up. Lots of dead code too.